### PR TITLE
feat: allow user templates

### PR DESCRIPTION
**Pull request by ZimGil**
## Feature: User Templates
* Users can now write their own templates for Tweets
  > Must add an `tweet` string under `announcements` object to `package.json`
  > Variables string as: `<%= variableName %>`

* Possible variables:
  * Package name: `<%= package %>`
  * Version number: `<%= version %>`
  * Package description: `<%= description %>`
  * Package author: `<%= author %>`
  * Homepage link: `<%= homepage%>`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,6 +28,9 @@
     "url": "https://github.com/Kibibit/announce-it/issues"
   },
   "homepage": "https://github.com/Kibibit/announce-it#readme",
+  "announcements": {
+    "tweet": "Version <%= version %> of <%= package %> is live! check it out <%=homepage %>"
```
Comment (Thatkookooguy) [inline package.json]: nit: newline (`\n`) before the homepage link

File: src/index.ts [hunk 2]
```diff
@@ -20,13 +19,22 @@ const client = new Twitter({
 client
   .get('account/verify_credentials')
     .then(() => fs.readJson(`${root}/package.json`, 'utf8'))
-    .then((packageData) => {
-      return `Version ${packageData.version} of ${packageData.name} is out!`;
-    })
+    .then((packageData) => getTweet(packageData))
```
Comment (Thatkookooguy) [inline src/index.ts]: get makes it look like it was not generated. maybe one of those?
 `generateTweet` | `createTweet` | `renderTweet` ?

File: package.json (hunk 1 above)
Comment (ZimGil) [reply package.json]: 1. This is user responsibility
2. Actually the link turns into a media

File: src/index.ts (hunk 2 above)
Comment (ZimGil) [reply src/index.ts]: Going with `generateTweet`

File: package.json (hunk 1 above)
Comment (Thatkookooguy) [reply package.json]: yup. this repository is the user in this case :-)

**Pull request merged**